get-booking/test/test.js

```diff
@@ -2,18 +2,19 @@ const assert = require('assert');
 const GetBookingService = require('../src/get-booking-service');
 
 describe('GetBooking Service Tests', () => {
-	const getBookingService = new GetBookingService();
 
 	it('returns a booking when a valid driving licence number is provided', () => {
-		getBookingService.drivingLicenceNumber = '32519228';
-		const data = getBookingService.getBooking();
-		assert.equal(true, (data.length === 1));
+		GetBookingService.getBooking('AAAAA000000AA0AA', '', (response) => {
+			assert.equal(true, (response.length === 1));
```

Comment:
"Blue is the sky" anti-pattern.

Parameters of assert.equal are ( actual, expected, message? )

Therefore:

assert.equal(data.length, 1);

Reply:
Done